src/Files.App/ServicesImplementation/Settings/UserSettingsService.cs

```diff
@@ -108,7 +101,7 @@ public override bool ImportSettings(object import)
         private TSettingsService GetSettingsService<TSettingsService>(ref TSettingsService settingsServiceMember)
             where TSettingsService : class, IBaseSettingsService
         {
-            settingsServiceMember ??= Ioc.Default.GetService<TSettingsService>();
+            settingsServiceMember ??= Ioc.Default.GetService<TSettingsService>()!;
```

Comment:
What's this change for?

Reply:
Remove nullable annotation (`Ioc.Default.GetService()` might return null)